src/server/project.ts

```diff
@@ -142,6 +154,35 @@ namespace ts.server {
             return this.cachedUnresolvedImportsPerFile;
         }
 
+        public static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void): {} {

```

Comment:
I considered this but didn't think it was worthwhile given that we'd have to mock out a `CompilerOptions` guaranteed to behave the same as NodeJS's resolution (which still differs from our own given how we crack open package.jsons and other stuff). I'm actually going to switch this to just use `require` directly anyway now that I've learned how to set up a different initial search path.